controllers/api/index.js

```diff
@@ -4,6 +4,10 @@ const userRoutes = require('./user-routes');
 const forumRoutes = require('./forum-routes')
 
 router.use('/users', userRoutes);
+<<<<<<< HEAD
+router.use('/forum', forumRoutes)
+=======
 router.use('/forum', forumRoutes);
+>>>>>>> dde5ae0195c07015e51679b2a1fa2a56fe2f6da9
 
```

Comment:
Lines 7 - 11 Unresolved Merge conflict?

Reply:
I think so, we can just delete that, and commit it with the next push.